README.md

```diff
@@ -3,18 +3,32 @@
 This is the main website for a pledge to stop supporting Python 2 for free in
 open source software.
 
+# Preview the changes
+
+If you are lazy, just send a PR. The continuous integration will build a preview
+on  a netlify subdomain. Once the PR is submitted, look in the continuous integration
+section for the `Netlify`, you may need to click 'Show all checks' section and click on `details`.
+
+![](prev.png)
+
 # Run locally
 
-Install Jekyll : `gem install jekyll`, `gem install github-pages`
+Install Jekyll, we recommend installing bundler first: `gem install bundler`
```

Comment:
The first part of this doesn't quite read right. I'd drop the mention of Jekyll here.